src/PublishPort.jsx

```diff
@@ -47,7 +91,13 @@ export const PublishPort = ({ id, item, onChange, idx, removeitem, itemCount })
                             min={1}
                             max={65535}
                             value={item.hostPort || ''}
-                            onChange={(_event, value) => onChange(idx, 'hostPort', value)} />
+                            validated={validationFailed?.hostPort ? "error" : "default"}
+                            onChange={(_event, value) => {
+                                utils.validationClear(validationFailed, "hostPort", onValidationChange);
+                                utils.validationDebounce(() => onValidationChange(validatePublishPort(value, "hostPort", validationFailed)));
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1354-20230912-120439-9f74a06a-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)